# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/gcc/gcc33/libstdc++-localedir.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

pass the install location for locales (datadir) to subdirectories

--- gcc-3.3.3/libstdc++-v3/Makefile.in	2003-07-04 21:53:54.000000000 +0200
+++ gcc-3.3.3/libstdc++-v3/Makefile.in-p	2004-11-04 01:34:50.772314280 +0100
@@ -148,6 +148,8 @@
 MAINT_CHARSET = latin1
 
 SUBDIRS = include libio libmath libsupc++ src po testsuite 
+SUBDIR_FLAGS_TO_PASS = \
+	"datadir=$(datadir)"
 
 mkinstalldirs = $(SHELL) $(toplevel_srcdir)/mkinstalldirs
 
@@ -294,7 +296,7 @@
 	  else \
 	    local_target="$$target"; \
 	  fi; \
-	  (cd $$subdir && $(MAKE) $(AM_MAKEFLAGS) $$local_target) \
+	  (cd $$subdir && $(MAKE) $(AM_MAKEFLAGS) $(SUBDIR_FLAGS_TO_PASS) $$local_target) \
 	   || case "$$amf" in *=*) exit 1;; *k*) fail=yes;; *) exit 1;; esac; \
 	done; \
 	if test "$$dot_seen" = "no"; then \
